ios/engine/KMEI/KeymanEngine/Classes/Extension/UserDefaults+Types.swift

```diff
@@ -18,7 +19,9 @@ public extension UserDefaults {
     do {
       return try array.map { try decoder.decode(InstallableKeyboard.self, from: $0) }
     } catch {
-      SentryManager.captureAndLog(error, message: "Error decoding keyboards: \(error)")
+      let message = "\(String(describing: error))"
```

Comment:
This message seems to have diverged.

I wonder about the verbosity of this logging pattern -- previously we managed to keep it to one line which was _great_ but now we have three lines, which makes it harder to keep consistent and clean, as well as being a lot noisier to read

Reply:
The focus for now is just to eliminate XCGLogger, but we can revisit the design of logging later and may be able to do better with access to iOS 14 APIs.